api/accounts_api_test.py

```diff
@@ -26,36 +26,44 @@
 from api import accounts_api
 from api import register
 from internals import models
+from google.cloud import ndb
+
+client = ndb.Client()
 
 
 class AccountsAPITest(unittest.TestCase):
 
   def setUp(self):
     self.app_admin = models.AppUser(email='admin@example.com')
     self.app_admin.is_admin = True
-    self.app_admin.put()
+    with client.context():
```

Comment:
These with-statements look correct, but they do get kind of repetitive.  Would it work to override the run() method in each test class like this?

```
  def run(self, result=None):
    with client.context():
      super(AccountsAPITest, self).run(result=result)
```

Reply:
Yes, it seems to work fine. Also, as [suggested](https://github.com/GoogleChrome/chromium-dashboard/pull/1365#issuecomment-866246577) by you earlier, I have made a base class for the tests (See [these](https://github.com/GoogleChrome/chromium-dashboard/blob/718bff6e47091cb73159c37c2e87c16d3094ea45/testing_config.py#L128) lines). I have changed the baseclass of all the tests in [this](https://github.com/GoogleChrome/chromium-dashboard/pull/1365/commits/718bff6e47091cb73159c37c2e87c16d3094ea45) commit

The tests work fine. Thank you for pointing out this method. I was unaware about this method as I have never used unittest before in any of my projects.